pages/send/Send.tsx

```diff
@@ -101,23 +109,16 @@ export function Send() {
             lnurlDetails.minSendable === lnurlDetails.maxSendable &&
             !lnurlDetails.commentAllowed
           ) {
-            try {
```

Comment:
Is it ok toremove the try-catch? what happens if there is an exception now?

Reply:
This was actually buggy, since there's already a `try catch` outside which can take care of this error (if any). Also because it was being handled by this catch it returns `true` instead of `false`, so it should be good after this removal